sources/platform/console/billing.md

```diff
@@ -0,0 +1,60 @@
+
+---
+
+## Current Period
+
+The **Current period** tab is a comprehensive resource for understanding your platform usage during the ongoing billing cycle. It provides an overview of your total usage and indicates the start and end dates of the current billing period.
+
+In addition to this, the tab features a **Plan Consumption** Graph. It shows how much of your free or paid plan has been utilized up to this point. The graph offers a broad perspective of your overall usage.
+
+Apart from the platform usage breakdown graph, this tab provides more detailed information. You can explore the sections under the graph. You can access detailed statistics related to **Actors**, **Data transfer**, **Proxy**, and **Storage**.
+
+![Apify Console current period view](./images/console-billing-current-period.png)
+
+## Historical Usage
+
+The **Historical usage** tab provides a detailed view of your monthly platform usage, excluding any free Actor compute units or discounts from your subscription plan. You can navigate through the months either by selecting from a dropdown list or by using arrows to move one month backward or forward.
+
+The tab features an adjustable bar chart. This chart can be customized to display statistics either on a monthly or daily basis. Additionally, you have the option to view these statistics as either absolute or cumulative numbers, providing flexibility in how you analyze your usage data.
+
+![Apify Console historical usage view](./images/console-billing-historical-usage.png)
+
+Below the bar chart, there is a table titled **Usage by Actors**. This table presents a detailed breakdown of the Compute units used per Actor and the associated costs. It clearly explains how each Actor contributes to your overall platform usage and expenses.
+
+![Apify Console historical usage by actor view](./images/console-billing-historical-usage-by-actors.png)
+
+## Subscription
+
+The **Subscriptions** tab offers a central location to manage various aspects of your subscription plan. Here, you can access details about your current plan and explore options to upgrade to a different one.
+
+This tab also allows you to manage your payment methods. Additionally, you can adjust your billing details to ensure they remain up-to-date.
+Another feature of this tab is the visibility of any special offers currently applied to your account.
+
+![Apify console subscription view](./images/console-billing-subscription.png)
+
+## Pricing
+
+The **Pricing** tab offers a way to quickly check all pricing related to Apify usage for your account.
+
+![Apify console pricing view](./images/console-billing-pricing.png)
+
+## Invoices
+
+The **Invoices** tab is where you can find your current and previous invoices for Apify platform usage.
+
+![Apify console invoices view](./images/console-billing-invoices.png)
+
+## Limits
+
+The **Limits** tab showcases the limits of Apify platform usage that are based on your current subscription plan.
```

Comment:
Here you can add that some of them are customizable (hard platform monthly limit on which you'd be blocked - this prevents you from burning more that you want, and data retention limit).